harvdev_utils/chado_functions/synonym.py

```diff
@@ -44,11 +44,10 @@ def synonym_name_details(session, synonym_name):
         unicode version -> 'Hsap\\00005-α-<up>001</up>'
     """
     pattern = r"""
-        ^([A-Z]:){0,1}  # May have T: or not {0 or 1} Not sure of variety so any captial letter is fine
-        ([A-Z]{1}       # 1 Capital letter
-        [a-z]{3})       # 3 lower case letters
-        \\              # forward slash
-        (.*)            # anything else
+        ^([A-Z]:){0,1}   # May have T: or not {0 or 1} Not sure of variety so any captial letter is fine
```

Comment:
Just had a look at this. This revised regex on organism.abbreviation used at the beginning of non-Dmel gene symbols may still be too strict. I sent you a separate e-mail with a query for non-standard abbreviations used for genes and tools.

Reply:
I could go for 
^([A-Z]:){0,1}
\S+
\\
(.*)

But I was worried about names that might have a '\' in them?

Reply:
Ah it does not matter if they do. I can lookup the species and if it does not exist then it is not a species prefix. Doh!